awx/ui_next/src/screens/Inventory/InventoryHosts/InventoryHostList.test.jsx

```diff
@@ -293,4 +304,33 @@ describe('<InventoryHostList />', () => {
     });
     await waitForElement(wrapper, 'ContentError', el => el.length === 1);
   });
+  test('should render enabled ad hoc commands button', async () => {
```

Comment:
Hey @AlexSCorey, do you want to remove the best below?

Reply:
@nixocio Yes, thanks for catching that.  Any other changes requested?